src/lib/configuration.ts

```diff
@@ -198,35 +195,6 @@ export class Configuration {
         request.headers['Authorization'] = `Bearer ${
           Configuration.accessKeys.oAuthTokens.accessToken
         }`;
-      } else {
```

Comment:
This part should be kept with `else if (Configuration.accessKeys.updateOAuthFn) {`

Reply:
But inside the block what i should do?? because we don't have anymore the registerTokens method.